extension/js/common/xss_safe_factory.ts

```diff
@@ -211,6 +211,14 @@ export class XssSafeFactory {
     return `<div class="${this.destroyableCls} reply_message_button"><img title="Secure Reply" src="${this.srcImg('svgs/reply-icon.svg')}" /></div>`;
   }
 
+  btnEndPPSession = () => {
+    return `<div class="finish_session gb_qe gb_oe" data-test="finish-session">
```

Comment:
Also please add a title so that the encrypted inbox icon has it, and double check it doesn't clash in Gmail.

Reply:
I add the title here:
```TS
$(this.factory.btnEndPPSession())
.attr(this.webmailName === 'gmail' ? 'data-tooltip' : 'title', 'End Current Session')
...
```